spec/requests/api/v1/assignment_details_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+describe Api::V1::AssignmentDetailsController, type: :request do
+  let(:split) { FactoryGirl.create(:split, name: 'excellent_feature', registry: { enabled: 50, disabled: 50 }, location: 'Home page') }
+
```

Comment:
this needs some basic auth, see [here](https://github.com/Betterment/test_track/pull/43/files#diff-fb80641b93ea2ec5501b64f18b9d2d02R9) for an example